lib/app/ctx.ex

```diff
@@ -581,4 +614,23 @@ defmodule App.Ctx do
   def change_timer(%Timer{} = timer) do
     Timer.changeset(timer, %{})
   end
+
+  alias App.Ctx.Session
```

Comment:
I'm increasingly not a fan of the "Contexts" in Phoenix.
They don't help to _organise_ code, they simply dump all code in to `Ctx` 
that _should_ be in the same file as the schema or controller.

Not a problem with _this_ PR more something that I'm going to _fix_ in a future refactor. 💭